test/benchee_test.exs

```diff
@@ -804,6 +804,23 @@ defmodule BencheeTest do
     end
   end
 
+  describe "reduction measurement" do
+    test "measures reduction count when instructed to do so" do
+      output =
+        capture_io(fn ->
+          Benchee.run(
+            %{"To List" => fn -> Enum.to_list(1..100) end},
+            Keyword.merge(
+              @test_configuration,
+              reduction_time: 2
```

Comment:
again, could we reduce the time here please? :D 2 seconds is almost the time the entire suite takes for me locally :sweat_smile: